<% if resource.errors.any? %>
  <div id='error_explanation'>
    <div class='alert alert-danger'>
      入力欄に<%= resource.errors.count %>個のエラーがあります。
    </div>
    <ul>
      <% resource.errors.full_messages.each do |message| %>
        <li><%= message %></li>
      <% end %>
    </ul>
  </div>
<% end %>
